docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,69 @@
 [//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
 
+# Setup local deployment for mobile phone connections
+
+To allow a mobile phone to connect to your local deployment, you will need to make the following configuration steps.
+
+1. Have your project opened in the Unity Editor.
+1. Open the configuration window through **SpatialOS** > **GDK Tools configuration**.
+1. Fill your local machine's IP into the **Runtime IP for local deployment** field.
+1. Click **Save**, and close the window.
+
+From this point on, all local deployments launched through the editor will allow your mobile phone to connect to it.
+
```

Comment:
```
# Prepare your project to connect to a local deployment

To connect your Android device to a local deployment, follow these steps:

 1. Open your project in the Unity Editor.
1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
1. Fill your [local machine's IP](This needs a link on how to find it) into the **Runtime IP for local deployment** field.
1. Click **Save**, and close the window.

This ensures that any local deployment that is launched via the Unity Editor is set up correctly to connect to your Android device. 
```

always try to use active voice, Unity Editor is always capitalized.

Reply:
<3

Reply:
+1 on Jess's suggestions. A couple of additions...

1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
...
These steps ensure that any local deployment you launch via the Unity Editor is set up to connect to your Android device correctly.